buildtest/menu/build.py

```diff
@@ -147,16 +214,22 @@ def func_build_subcmd(args):
                 result = executor.dry_run(builder)
 
     if not args.dry:
-        print
-        print
-        print("==============================================================")
-        print("                         Test summary                         ")
+        print("\n")
+        print("\n")
+        print("{:=<60}".format(""))
```

Comment:
Let's also combine some of these statements, you can add the \n to have more than one per print().

Reply:
makes sense. Fixed now